boogiestats/templates/boogie_ui/root.html

```diff
@@ -9,6 +9,8 @@
               content="A pass-through proxy for groovestats.com that records non-ranked song scores." />
         <link rel="stylesheet" href="{% static "boogie_ui/vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css" %}" />
         <link rel="stylesheet" href="{% static "boogie_ui/boogiestats.css" %}" />
+        <link href="https://fonts.cdnfonts.com/css/miso" rel="stylesheet" />
+        <link href="https://fonts.cdnfonts.com/css/wendy" rel="stylesheet" />
```

Comment:
I'd rather avoid basing on external resources. Could we vendor these fonts? What are the licenses?

Reply:
Miso license, source:fontsquirell
L I C E N S E   I N F O R M A T I O N
———————————————————-
You can use MISO for free.
You can share MISO with your friends
as long as you include this text file.
You are NOT permitted to sell MISO.

As for wendy: could by unlicensed as most pages skip that part but one page mentions:
EN: Only available for testing the font. Non-commercial and reproduction prohibited. Please contact the author for official purchase of use.
Could be boilerplate, needs doublecheck